### ISSUE TITLE
Error in Finetuning script

### ISSUE THREAD

**Issue by HITchina** [AutoLab-SAI-SJTU/InfiniteVGGT#12]
Hello Dear! Your research work is inspiring for the Computer Vision Society. I am trying to finetune the model on MVS_synth dataset. when I run the script it gives an error from the Losses script can you please help me to resolve this issue?

<img width="965" height="38" alt="Image" src="https://github.com/user-attachments/assets/6194ab33-5369-4bca-8ddd-a3f694dd5189" />

**Comment by Henryyuan429** [comment]
Hi! Thank you for your kind words and interest in our work.

Regarding the error you're encountering, please note that our proposed method is entirely training-free and the loss scripts haven't been actively used or tested for this specific pipeline.

**Comment by HITchina** [comment]
@Henryyuan429  Hello, dear hope you are fine and doing well. I want to ask a question regarding the original VGGT training model. Is it possible to train the original VGGT model on your provided training script? thanks
